### ISSUE TITLE
[Screen Reader-Help-Hardware] Link purpose is not descriptive enough for the ”Here” link on hardware pane.

### ISSUE THREAD

**Issue by v-swasha** [Microsoft/pxt#2546]
**User Experience:**
If a link doesn't have valid text to identify itself, then screen readers will not be able to read the link clearly, it will just be read out to the visually impaired user as “link.” Which then makes it difficult for the blind users to understand the purpose or meaning on the link.

**Test Environment**
OS: RS2 Version 1703(OS Build 15063.483)
Platform: Edge.
Screen reader: Narrator

**Repro Steps**
1-Navigate to https://makecode.microbit.org/acc
2-Navigate to the Micro bit section element and select Code control given on it.
3-Navigate to the Help control lying in the header section on the page and select it.
4-Navigate to 'Hardware' control and select it.
5-Navigate to the various controls lying in the pane opened.
6-Verify, Here link on hardware pane is descriptive enough or not.

**Actual Result**
Link purpose is not descriptive enough for the ”Here” link on hardware pane. Due to this narrator is reading only here link.

**Expected Result**
Link Purpose should be descriptive for the 'Here' link. It should clearly provide the purpose of the link for example 'Uploading programs to your micro: bit’ here link.

**Note:-**
Same issue exists for LEDs and pins and USB links on hardware pane.

**MAS Reference**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={06a891af-210c-4c6d-bb3e-92babab209e7}

Suggested fix:-
1-Providing link text that describes the purpose of a link or a link for anchor elements
e.g.
<a href=routes.html>
  Current routes at Boulders Climbing Gym
</a>
a href="routes.html
 img src="topo.gif" alt="Current routes at Boulders Climbing Gym" /> 
/a
2-Providing text alternatives for the area elements of image maps
3-Using scripts to change the link text (choose short or long link text )
4- Use 'aria-labelledby' or 'aria-label' for link purpose 

**Key References for Fix:** 
http://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-refs.html 

**Please refer attachment for more details:**
[MAS2.4.4_Hardware_Links purpose is not descriptive enough for all links on hardware.zip](https://github.com/Microsoft/pxt/files/1179660/MAS2.4.4_Hardware_Links.purpose.is.not.descriptive.enough.for.all.links.on.hardware.zip)

**Comment by veler** [comment]
Fix available on [https://makecode.microbit.org/](https://makecode.microbit.org/)

**Comment by v-mesala** [comment]
Verified on URL: https://makecode.microbit.org/acc and issue is fixed now. Hence closing the bug
